ts/react-responsive-components.tsx

```diff
@@ -134,7 +134,7 @@ export interface ResponsiveProps extends MixedInHoCProps {
 	// https://github.com/DefinitelyTyped/DefinitelyTyped/pull/56210
 	// We refine children type as the following because in `Responsive` component's option 2, we are able
 	// to pass a function as the children of the Responsive component.
-	children?: React.ReactNode | undefined | ((responsiveKey: string) => JSX.Element)
+	children?: React.ReactNode | undefined | ((responsiveKey: any) => React.JSX.Element | null)
```

Comment:
What happened? I thought this was working with strings.

Also, doesn't the `js/react-responsive-components.d.ts` file need to be updated if we update this here?

Reply:
normal type check works, but strict null check doesn't like it 😂

Reply:
Is the strict null check okay with this typedef?
```suggestion
	children?: React.ReactNode | undefined | ((responsiveKey: string) => React.JSX.Element | null)
```